TestRunner.psm1

```diff
@@ -21,6 +20,10 @@ function Get-ServerMetrics {
 
     }
 
+    $expectedPerformanceCounters = @("\SQLServer:Access Methods\Full Scans/sec", "\SQLServer:SQL Statistics\Batch Requests/sec", "\SQLServer:General Statistics\User Connections", "\ASP.NET\Requests Queued", "\SQLServer:Wait Statistics(*)\Network IO Waits", "\SQLServer:Wait Statistics(*)\Page IO Latch Waits")
+    $actualPerformanceCounters = Compare-Object (typeperf -q) $expectedPerformanceCounters -PassThru -IncludeEqual -ExcludeDifferent
```

Comment:
Is this saying that we have a hard-coded list on line 23, but line 24 then removes any of those items that aren't available?

Reply:
I answered my own question: yes.

Reply:
yes. some of the logic was lost during refactoring so counters ran into issues on the database server while querying what was not available.